src/components/WrapperLastSeen.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect, useContext } from 'react';
+import { SettingsBarContext, SettingsBarContextType } from './SettingsBarContext'
+
+import { ThemeContext, ThemeContextType } from './ThemeContext';
+
+import Title from './Title';
+import Container from './Container';
+import Slider from './Slider';
+import SmallInfoCard from './SmallInfoCard';
+
+import api from '../service/api';
+
+type MovieId = {
```

Comment:
I remember I left comment regarding such type - we have one same  already and can reuse it